pages/linux/dnf.md

```diff
@@ -0,0 +1,24 @@
+# dnf
+
+> RHEL, Feodra, and CentOS package management utility (Replaces yum)
```

Comment:
Reordering this sentence makes more sense IMO: "Package management utility for RHEL, Feodra, and CentOS (replaces yum)"